psycop/projects/scz_bp/feature_generation/eligible_prediction_times/scz_bp_cohort_loader.py

```diff
@@ -0,0 +1,78 @@
+from typing import Callable
+
+import polars as pl
+
+from psycop.common.cohort_definition import Cohort, FilteredCohort, StepDelta
+from psycop.common.feature_generation.loaders.raw.load_visits import ambulatory_visits
+from psycop.projects.scz_bp.feature_generation.eligible_prediction_times.single_filters import (
+    add_age,
+    excluded_by_washin,
+    max_age,
+    min_age,
+    min_date,
+    washout_move,
+    without_prevalent_scz_or_bp,
+)
+from psycop.projects.scz_bp.feature_generation.outcome_specification.first_scz_or_bp_diagnosis import (
+    get_first_scz_bp_diagnosis_after_washin,
+)
+
+
+class SczBpCohort(Cohort):
+    def __init__(self):
+        self.stepdeltas: list[StepDelta] = []
```

Comment:
We collect the stepdeltas in two places now - in FilteredCohort and in self.stepdeltas. I would make this private, and encourage using from FilteredCohort.


```suggestion
        self.stepdeltas: list[StepDelta] = []
```